src/static/mermaid.html

```diff
@@ -22,6 +22,9 @@
           mermaid.initialize({ ...config, startOnLoad: false });
           mermaid.render('mermaid-svg', definition, (svgCode) => {
             container.innerHTML = svgCode;
+            const style = document.createElement('style');
+            style.innerText = '@import url("../../node_modules/@fortawesome/fontawesome-free/css/all.min.css");@import url("../../node_modules/@fortawesome/fontawesome-free/css/v4-shims.min.css");';
```

Comment:
```suggestion
            style.innerText = '@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css");';
```

Relative path won't work here since marmaid.ink does not serve font-awesome library publicly.